modules/understanding-telemetry-and-insights-operator-data-flow.adoc

```diff
@@ -7,7 +7,7 @@
 
 The Telemeter Client collects selected time series data from the Prometheus API. The time series data is uploaded to api.openshift.com every four minutes and thirty seconds for processing.
 
-The Insights Operator gathers selected data from the Kubernetes API and the Prometheus API into an archive. The archive is uploaded to link:https://cloud.redhat.com[cloud.redhat.com] every two hours for processing. The Insights Operator also downloads the latest Insights analysis from link:https://cloud.redhat.com[cloud.redhat.com]. This is used to populate the *Insights status* pop-up that is included in the *Overview* page in the {product-title} web console.
+The Insights Operator gathers selected data from the Kubernetes API and the Prometheus API into an archive. The archive is uploaded to link:https://console.redhat.com[console.redhat.com] every two hours for processing. The Insights Operator also downloads the latest Insights analysis from link:https://console.redhat.com[console.redhat.com]. This is used to populate the *Insights status* pop-up that is included in the *Overview* page in the {product-title} web console.
```

Comment:
@radekvokal can you please review this review this one line? I assume that the Insights Operator uses console.redhat.com now for the archiving and downloads, but I think it is useful to double-check with you. Thanks!